resources/destination-types/elasticsearch.yaml

```
@@ -13,12 +13,20 @@ spec:
       type: strings
       default: []
```

```suggestion
      default: []
      options:
          gridColumns: 12
```

Let's use gridColumns to make the endpoints wider and put the two indexes next to each other.